sesame2spiner/io_eospac.hpp

```diff
@@ -75,10 +75,8 @@ class Bounds {
       // should be single-precision epsilon b/c that's whats used for the logs
       constexpr Real epsilon = std::numeric_limits<float>::epsilon();
       const Real min_offset = 10*std::abs(epsilon);
-      if (min < 0) offset = std::abs(min) + min_offset;
-      else if ( min == 0 ) {
-        offset = min_offset;
-      }
+      if (min <= 0) offset = 1.1*std::abs(min) + min_offset;
```

Comment:
Could I get more context here? Would it be worth commenting in the code how offset is calculated and why?

Reply:
Added a comment. Basically the issue was that the code was choosing an offset for min < 0 to bring min to identically zero, which still doesn't work when you take the log. This situation now handles min == 0 with min_offset and min < 0 with the 1.1 so that you're a little away from identically zero after shifting.